### ISSUE TITLE
The search method stopped working

### ISSUE THREAD

**Issue by bumsun** [facundoolano/google-play-scraper#280]
* Operating System: Ubuntu 16.04
* Node version: v8.3.0
* google-play-scraper version: 6.4.1 

### Description: 
The search method stopped working.

### Example code:
```js
gplay.search({
                term: req.body.query,
                num: 250,
                country: req.body.country,
                lang: "ru"
            })
```

### Error message:
```js
"err":{"status":404}
```

**Comment by Brabys2012** [comment]
too

**Comment by yurganov** [comment]
Very quick fix 
https://github.com/facundoolano/google-play-scraper/pull/281

**Comment by yurians19** [comment]
the same error happens to me!

**Comment by yurians19** [comment]
How did you solve?

**Comment by KaKi87** [comment]
Hello,

I have this error :

```
{ Error: App not found (404)
    at /opt/KPlaySearch/node_modules/google-play-scraper/lib/utils/request.js:42:19
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7) status: 404 }
```

Is it the same problem ?
I was going to create a new issue but... 🙃 

If it is the same, then there's the fix, but even if I replace the package's version to `*` in `package.json` and updating the modules with `yarn install`, the problem is still there.

Thanks.

**Comment by bumsun** [comment]
Is the version with fix still not available in npm?

**Comment by facundoolano** [comment]
This should be fixed in v6.2.2
